File: Floyds Triangle/printFloydsTriangle.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

// Function to print star pattern.
function printFloydsTriangle(n) {
  let count = 1;
  for (let i=1; i<=n; i++) {
    let pattern = "";
    for (let j=1; j<=n; j++) {
      if (j <= i) {
        if (count < 10) {
          pattern += `${count}  `;
        }
        else {
          pattern += `${count} `;
        }
        count++;
      }
      else {
        pattern += " ";
      }
    }
    console.log(pattern);
  }
}

// value of n from user and calling printFloydsTriangle on them.
rl.question('Enter the value of n : ', n => {
  printFloydsTriangle(Number(n));
  rl.close();
});

File: Floyds Triangle/printFloydsTriangle.py
n = int(input("Enter the value of n : "))
count = 1

for i in range(1, n+1):
  for j in range(1, n+1):
    if j <= i:
      if count < 10:
        print(str(count) + "  ", end = "")
      else:
        print(str(count) + " ", end = "")
      count += 1
    else:
      print(" ", end = "")
  print()

